### ISSUE TITLE
In the process of connecting with PLC, "Forward Open" issue.

### ISSUE THREAD

**Issue by joon14-lee** [EIPStackGroup/OpENer#362]
Hi,

OpEner is being used by using Mingw in Windows 10.
At this time, we tried to connect with LS Electronics' PLC as a client. At this time, as a result of checking through wireshark, it was confirmed that Forward Open failed.

However, I left a question because it was difficult to figure out the exact reason why it failed.

The contents in wireshark are as follows.

![image](https://user-images.githubusercontent.com/83627718/153521832-23c8d1ec-eb83-4641-9371-90c9a0d44985.png)

Connection Manager - Forward Open

![image](https://user-images.githubusercontent.com/83627718/153525491-d8059685-8c78-40d8-8390-860c56d0ef53.png)
![image](https://user-images.githubusercontent.com/83627718/153525520-bc75e752-e564-4e06-a271-a8b9c790a702.png)

Connection failure

![image](https://user-images.githubusercontent.com/83627718/153525355-8de6b51a-e5f2-4f40-ad3a-13dfdfef8635.png)

**Comment by gongli-uestc** [comment]
Hi,

Did you solve this problem? I had the same problem as you！Forward Open failed!

**Comment by joon14-lee** [comment]
Hi.

I solved this problem. My problem was about the size of O->T data when requesting Fward Open. Problems are displayed in the wireshark capture content below.

![image](https://user-images.githubusercontent.com/83627718/156677346-e3cf113d-a41f-4e39-ab0b-fd3d802ff2e4.png)


There was a problem even though the number of O->T and T->O data on the PLC side and OpENer side was the same.
Since the settings on both sides were well done, I thought there would be a problem on the other side and modified the EDS file.
The problem was solved by deleting the red part of the picture below and the corresponding value that appears to determine the size of sending and receiving.

![image](https://user-images.githubusercontent.com/83627718/156677370-b1b8ec96-07da-4709-bc5c-46831a43e786.png)


The cause of the problem I thought might not be the actual cause, but this way I solved the problem.

I hope you solve the problem, too.
